doc/source/serve/handle-guide.md

```diff
@@ -46,6 +46,13 @@ The result of `deployment_handle.remote()` can also be passed directly as an arg
 :language: python
 ```
 
+:::{note}
+Different metrics are collected when Deployments are called
+via Python `ServeHandles` and when they are called via HTTP.
+
+See the [Ray Serve monitoring documentation](serve-monitoring) for more details.
+:::
+
```

Comment:
This is pretty noisy and likely not relevant to most readers of the section, can we make this a note at the top of monitoring docs where we talk about metrics instead?

Reply:
Yes, we were discussing this with @shrekris-anyscale and we thought that it might worth pointing out that the HTTP and Python calling are NOT made to be compatible. Currently the documentation makes to sound it so.
In the current version of the docs (the one online) the HTTP and Python calls are discussed in one single page - kind of like a comparison, which is why I was surprised to find out that the two behaves rather differently when it comes to metrics, after seeing nothing like that being mentioned in the section specifically talking about the two type of calls.

But fine by me, I will remove the notes from the Python `ServeHandle` and the HTTP call pages.